spec/catalog.yaml

```diff
@@ -102,7 +102,40 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/namespaces/{ns}:
+    parameters:
+      - $ref: '#/components/parameters/namespace'
+
+    get:
+      tags: [ Namespace ]
+      summary: Get information about a namespace
+      operationId: getNamespace
```

Comment:
nit: I just noticed that we have `CreateNamespace`, `listNamespaces` and `getNamespace`, we should probably make the capitalization consistent.

Reply:
OK, well, we choose the first letter capital case.